README.md

```diff
@@ -134,6 +134,12 @@ See [here](https://github.com/Azure/azure-functions-core-tools/blob/master/READM
 
 * [Python 3.6.x](https://www.python.org/downloads/)
 
+### PowerShell
+
+* [PowerShell Core 6.2 or higher](https://github.com/powershell/powershell#get-powershell)
+* [VS Code Debugger for PowerShell](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell)
+* [.NET CLI](https://docs.microsoft.com/dotnet/core/tools/?tabs=netcore2x)
```

Comment:
Is this just for extensions? If so - func cli is removing that requirement in favor of bundles - see https://github.com/Microsoft/vscode-azurefunctions/pull/1204

Reply:
No, the Azure Functions host does not give us the ability to run using a single, standalone executable. The PowerShell worker is a .NET console app essentially that must be started using:

```
dotnet ./workers/powershell/Microsoft.Azure.Functions.PowerShellWorker.dll
```

so `dotnet` needs to be available on the path. We are working with them on how to make that experience better... but you must have the dotnet CLI to run the PowerShell worker via the Azure Functions Core Tools

Reply:
I've opened an official issue on this now - https://github.com/Azure/azure-functions-powershell-worker/issues/195